polaris-react/locales/ko.json

```diff
@@ -355,7 +355,7 @@
       "clear": "지우기"
     },
     "IndexFilters": {
-      "searchFilterTooltip": "검색 및 필터링(F)",
+      "searchFilterTooltip": "검색 및 필터링",
```

Comment:
This translation looks like it may have been missing a space. The space will be added by the new logic.